apps/store/src/blocks/PageBlock.tsx

```diff
@@ -22,7 +21,7 @@ export const PageBlock = ({ blok }: PageBlockProps) => {
           <StoryblokComponent blok={nestedBlock} key={nestedBlock._uid} />
         ))}
       </main>
-      <DiscountBannerTrigger />
+      <PageBannerTriggers blok={blok} />
```

Comment:
Why don't we pass just the announcement here instead?
```suggestion
      <PageBannerTriggers blok={blok.announcement} />
```